lib/rules/function-no-unknown/__tests__/index.js

```diff
@@ -70,6 +70,22 @@ testRule({
 	],
 });
 
+testRule({
+	skip: true,
+	ruleName,
+	config: true,
+	customSyntax: 'postcss-styled-syntax',
+
+	accept: [
+		{
+			code: 'const buttonFontSize = css`font-size: ${({ size }) => (size === "small") ? "0.8em" : "1em"};`;',
+		},
+		{
+			code: 'const buttonFontSize = css`border-radius: ${`calc(${token.radiusBase} + 2px)`};`;',
+		},
+	],
+});
+
```

Comment:
[suggest] I think this PR fix should focus on `isStandardSyntaxFunction()`, so what about reverting the test for `function-no-unknown`? See also my idea details for https://github.com/stylelint/stylelint/pull/6556#issuecomment-1375183292